### PR TITLE
chore: remove mentions of DP

### PR DESCRIPTION
**Pull request by ankur-arch**
This section removes mentions of DP.
Except for some parts in the error reference, which I intend to remove after DP is sunsetted.

### CODE REVIEW

File: content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx
Comment (jharrell) [inline content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]:
```suggestion
Connection poolers like [Prisma Accelerate](https://www.prisma.io/data-platform/accelerate) and PgBouncer prevent your application from exhausting the database's connection limit.
```

File: content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx
Comment (jharrell) [inline content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]: Not sure, but maybe we should link to PDP docs here instead of the console route?

**Review by jharrell — APPROVED** [review]

File: content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx
Comment (ankur-arch) [reply content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]: That's a fair point, @nikolasburk , @ruheni any suggestions?

Comment (ruheni) [reply content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]: I agree with Jon's comment

Comment (nikolasburk) [reply content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]: I'm actually not sure if we should merge this now or later when https://github.com/prisma/docs/pull/5501 has been merged... I think both options are troublesome, just not sure which one is more troublesome. Does anyone have thoughts on that?

Comment (ankur-arch) [reply content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]: @nikolasburk ping me after you've merged 5501, I think that approach will be easier. I'll put this task to blocked for now.

Comment (ankur-arch) [reply content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]:
```suggestion
[Prisma Accelerate](/data-platform/accelerate) is a managed external connection pooler and global cache built by Prisma that is integrated in the [Prisma Data Platform](/data-platform) and handles connection pooling for you.
```

**Comment by jharrell** [comment]
Looks like this was removed in a different PR. Closing

**Pull request closed without merging**
